broker-daemon/broker-rpc/orderbook-service/get-orderbook.js

```diff
@@ -1,10 +1,13 @@
 const nano = require('nano-seconds')
+const MarketEventOrder = require('../../models/market-event-order')
```

Comment:
I think you should destructure `SIDES` from this since it's the only property you are using, e.g.

```
const { SIDES } = require('../../models/market-event-order')
```

also - is there a way to do this that doesn't require the rpc to reach into the models directly?